python/tests/experimental/core/test_udf_schema.py

```diff
@@ -319,3 +319,12 @@ def test_schema_name() -> None:
     assert "add5" not in bob_view.get_columns()
     assert "bob" in bob_view.get_columns()
     assert "udf" in bob_view.get_column("schema.col1").get_metric_names()
+
+
+def test_schema_list() -> None:
+    schema = udf_schema(schema_name=["", "bob"])  # maybe always include "" ?
```

Comment:
Discussed: let's include empty string 'global' namespace by default, but maybe an optional param that controls if the empty string namespace is included lets callers control this behavior while being hopefully not too surprising by default.